addons/stock_account/models/digest.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+
+class Digest(models.Model):
+    _inherit = 'digest.digest'
+
+    kpi_stock_valuation_total = fields.Boolean('Total Inventory Valuation')
+    kpi_stock_valuation_total_value = fields.Monetary(compute='_compute_kpi_stock_valuation_total_value')
+
+    def _compute_kpi_stock_valuation_total_value(self):
+        for record in self:
+            start, end, company = record._get_kpi_compute_parameters()
+            self.kpi_stock_valuation_total_value = self.env['stock.valuation.layer'] \
+                .read_group([('company_id', '=', company.id), ('create_date', '>=', start), ('create_date', '<', end)],
+                            ['value:sum'], [])[0]['value']
```

Comment:
Never saw this writing (a void list then [0]). Is that standard ?

Reply:
Yes I think, I saw a lot of code like this. But I've added a check in case it returns an empty list.